.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 repos:
   - repo: .
-    rev: v0.7.3
+    rev: v0.7.5
```

Comment:
I guess this can't be 0.7.5 yet, until after we tag.

Reply:
Maybe it could? Is it even needed if it's pointing to itself?

Reply:
I tried in https://app.circleci.com/pipelines/github/lightspeed-hospitality/pre-commit-hooks/144/workflows/775441a0-5e70-40d8-95bd-9a94472ce6aa/jobs/123